src/main/webapp/js/project_repo_search.js

```diff
@@ -0,0 +1,35 @@
+Behaviour.specify(".searchable", 'searchableField', 200, function(el) {
```

Comment:
some documentation would help. for example, what is 200? Or a link to https://github.com/jenkinsci/jenkins/blob/master/war/src/main/webapp/scripts/behavior.js